pkg/webhook/controlplane/ensurer.go

```diff
@@ -336,8 +349,11 @@ func (e *ensurer) EnsureKubeletConfiguration(_ context.Context, _ gcontext.Garde
 		new.FeatureGates = make(map[string]bool)
 	}
 
-	new.FeatureGates["CSIMigration"] = true
-	new.FeatureGates["CSIMigrationAWS"] = true
+	if !versionutils.ConstraintK8sGreaterEqual127.Check(kubeletVersion) {
```

Comment:
That's not intuitive and it is harder to follow than simply using:
```suggestion
	if versionutils.ConstraintK8sLess127.Check(kubeletVersion) {
```

`it is not >= 1.27` is the same as `it is < 1.27`.